app/src/main/java/org/wikipedia/Constants.kt

```diff
@@ -21,6 +21,8 @@ object Constants {
     const val ACTIVITY_REQUEST_VOICE_SEARCH = 45
     const val ACTIVITY_REQUEST_WRITE_EXTERNAL_STORAGE_PERMISSION = 44
 
+    const val ANDROID_METRICS_PLATFORM_EVENT_PREFIX = "android.metrics_platform."
+
```

Comment:
is the best place for this?
I figured this prefix will be used across all the files where MP events will eventually be submitted, not just ArticleEvent